app/models/cyclist.rb

```diff
@@ -0,0 +1,19 @@
+class Cyclist < SimpleDelegator
```

Comment:
I don't think the existing name is bad, but `CyclistInvitation` seems more descriptive of what this represents.